app/templates/request/_view_extension.js.html

```diff
@@ -0,0 +1,114 @@
+<script type="text/javascript">
+$(document).ready(function () {
+    // Hides all other divs except for the first. Also hides previous button on the first div.
+    $(".extension-control .extension-divs").each(function (e) {
+        if (e != 0)
+            $(this).hide();
+        else
+            $("#previous").hide();
+    });
+
+    // Handles click events on the first next button
+    $("#next-1").click(function (e) {
```

Comment:
This should probably be extension-next-1 (specific to this section) next-1 may be used somewhere else